testsuite/features/init_clients/sle_buildhost.feature

```diff
@@ -0,0 +1,116 @@
+# Copyright (c) 2016-2020 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+Feature: Be able to bootstrap a Salt buildhost via the GUI
+
+@buildhost
```

Comment:
I would only write the tag once at the feature level, as it applies to all the scenarios

Reply:
It was argued that there might be a difference in those cases (tag at feature VS tag on each scenario): it was suggested that the later might be more verbose than the former; we never found the time to check this.

Reply:
Tag at the feature level, from the point of view of implementation it's just adding the tag in all the scenarios.
So there are no differences in terms of performance or logging. If we will require the tag in all the scenarios, imho better if we just only have once at the feature level.

Reply:
Let's discuss the convention to follow out of this PR; once we get an agreement we can open a dedicated PR for that (updating the documentation as well with the desired convention).

Reply:
Sure.